### DISCUSSION TITLE
Something unforeseen has happened which cannot be recovered from.

### DISCUSSION THREAD

**Discussion by Xenoflux-RP** [nefarius/HidHide#69] [Q&A]
![image](https://user-images.githubusercontent.com/92618013/160318840-d9ace6ab-251a-4c85-b8a6-ebe9a2ac3189.png)
Hi, i'm getting this error, i tried reinstalling everything, this used to work before can
but now i needed it again cause i got a new controller and it is showing me this error.
can i get a help on how to fix this error?

**Comment by Strych74** [comment]
Tried installing and running this software today and encountered the same problem.  Attempting to navigate to the Devices tab instantly results in the above error and exiting the program.  Have tried uninstalling/reinstall, repair, install as administrator, running app as administrator but all give the same result.

Event Viewer showed the following:
Log Name:      Application
Source:        Application Error
Date:          2/08/2023 10:44:11 PM
Event ID:      1000
Task Category: Application Crashing Events
Level:         Error
Keywords:      
User:          DAVE-PC\daveh
Computer:      Dave-PC
Description:
Faulting application name: HidHideClient.exe, version: 1.2.122.0, time stamp: 0x6453fb70
Faulting module name: ucrtbase.dll, version: 10.0.22621.608, time stamp: 0xf5fc15a3
Exception code: 0xc0000409
Fault offset: 0x000000000007f61e
Faulting process id: 0x0x365C
Faulting application start time: 0x0x1D9C54FBBA136EE
Faulting application path: C:\Program Files\Nefarius Software Solutions\HidHide\x64\HidHideClient.exe
Faulting module path: C:\Windows\System32\ucrtbase.dll
Report Id: b3f3d65f-6a2d-4ed4-816e-86d86907673e
Faulting package full name: 
Faulting package-relative application ID: 
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" Guid="{a0e9b465-b939-57d7-b27d-95d8e925ff57}" />
    <EventID>1000</EventID>
    <Version>0</Version>
    <Level>2</Level>
    <Task>100</Task>
    <Opcode>0</Opcode>
    <Keywords>0x8000000000000000</Keywords>
    <TimeCreated SystemTime="2023-08-02T14:44:11.2295519Z" />
    <EventRecordID>2402</EventRecordID>
    <Correlation />
    <Execution ProcessID="1580" ThreadID="2984" />
    <Channel>Application</Channel>
    <Computer>Dave-PC</Computer>
    <Security UserID="S-1-5-21-3884830466-836811678-3237196634-1002" />
  </System>
  <EventData>
    <Data Name="AppName">HidHideClient.exe</Data>
    <Data Name="AppVersion">1.2.122.0</Data>
    <Data Name="AppTimeStamp">6453fb70</Data>
    <Data Name="ModuleName">ucrtbase.dll</Data>
    <Data Name="ModuleVersion">10.0.22621.608</Data>
    <Data Name="ModuleTimeStamp">f5fc15a3</Data>
    <Data Name="ExceptionCode">c0000409</Data>
    <Data Name="FaultingOffset">000000000007f61e</Data>
    <Data Name="ProcessId">0x365c</Data>
    <Data Name="ProcessCreationTime">0x1d9c54fbba136ee</Data>
    <Data Name="AppPath">C:\Program Files\Nefarius Software Solutions\HidHide\x64\HidHideClient.exe</Data>
    <Data Name="ModulePath">C:\Windows\System32\ucrtbase.dll</Data>
    <Data Name="IntegratorReportId">b3f3d65f-6a2d-4ed4-816e-86d86907673e</Data>
    <Data Name="PackageFullName">
    </Data>
    <Data Name="PackageRelativeAppId">
    </Data>
  </EventData>
</Event>

**Comment by SirPrimalform** [comment]
Just here to add that I'm having exactly the same problem.

**Reply by nefarius** [reply]
Try [this build](https://buildbot.nefarius.at/builds/HidHide/v1.6.297.0/1.6.297.0/bin/Release/x64/HidHideClient.exe).

**Comment by kebian** [comment]
I'm having this problem today as well after adding an xbox controller to my system.  Rebooting fixed it for me - I can access the Devices tab now.

**Comment by Jobima1st** [comment]
well, seems this has something to do with the HidHide's Applications list, if it has orphan paths for .exe's that deleted or moved or whatever

the last update of ds4windws also say so
<img width="1232" height="406" alt="image" src="https://github.com/user-attachments/assets/17d46437-b39f-4f0a-b90e-4724e62d3a16" />
